dockerfiles/Dockerfile

```diff
@@ -6,6 +6,15 @@ RUN apk add --no-cache \
 
 FROM scratch
 LABEL "com.centurylinklabs.watchtower"="true"
+LABEL "org.opencontainers.image.url"="https://containrrr.dev/watchtower/" \
+      "org.opencontainers.image.documentation"="https://containrrr.dev/watchtower/" \
+      "org.opencontainers.image.source"="https://github.com/containrrr/watchtower" \
+      "org.opencontainers.image.licenses"="Apache-2.0" \
+# unsure about capitalization
+      "org.opencontainers.image.title"="watchtower" \
+      "org.opencontainers.image.description"="A process for automating Docker container base image updates." \
+# The version might be forgotten. The version could be extraced into a `ARG` before the initial `FROM` or removed.
+      "org.opencontainers.image.base.name"="alpine:3.19.0"
```

Comment:
Any feelings? If not I would tend towards removing the concrete version. It adds little in additional information but makes the Dockerfile somewhat more complex.

Reply:
You could be able to do something like
```Dockerfile
ARG BASE_IMAGE=alpine:3.19.0

FROM --platform=$BUILDPLATFORM $BASE_IMAGE as alpine
[..]
FROM scratch
ARG BASE_IMAGE
LABEL "org.opencontainers.image.base.name"="$BASE_IMAGE"
```

can't comment on if it would be useful in practice (can't recall ever looking for the labels myself)